src/app/core/entity/entity-mapper.service.ts

```diff
@@ -152,14 +152,13 @@ export class EntityMapperService {
     const rawData = entities.map((e) =>
       this.entitySchemaService.transformEntityToDatabaseFormat(e)
     );
-    entities.forEach((entity) => {
-      this.sendUpdate(entity, entity._rev === undefined ? "new" : "update");
-    });
     const results = await this._db.putAll(rawData);
     results
       .filter((res) => res?.ok)
       .forEach((res, idx) => {
```

Comment:
I think there will be a problem here with the `idx` as you might filter out some entities before.
I would recommend adding a test case where some entities fail to be saved.